src/System.CommandLine.Suggest/SuggestionStore.cs

```diff
@@ -41,10 +42,20 @@ public string GetCompletions(string exeFileName, string suggestionTargetArgument
                                      })
                 {
                     process.Start();
-
+                    process.BeginOutputReadLine();
+                    
+                    var stringBuilder = new StringBuilder();
+                    process.OutputDataReceived += (sender, eventArgs) =>
+                    {
+                        if (eventArgs.Data != null)
+                        {
+                            stringBuilder.AppendLine(eventArgs.Data);
```

Comment:
Thank you, might be the mistake is here! I hope it is my last. Otherwise I am completely at loss right now on why the test keeps faling.
Regarding Append/AppendLine: seems not the case here - I have tested it, the eventArgs.Data comes without any line terminator.